src/containers/bitset.c

```diff
@@ -53,9 +53,15 @@ bitset_container_t *bitset_container_create(void) {
     if (!bitset) {
         return NULL;
     }
-    // sizeof(__m256i) == 32
-    bitset->words = (uint64_t *)roaring_aligned_malloc(
-        32, sizeof(uint64_t) * BITSET_CONTAINER_SIZE_IN_WORDS);
+    if ( croaring_avx512() ) {
```

Comment:
I think that this check should be guarded with CROARING_IS_X64 otherwise you may get compilation failures on non-x64 systems (including 32-bit Windows).

Reply:
yes, i will add CROARING_IS_X64